packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -244,7 +258,7 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
   };
 
   private _onAChanged = (ev: React.MouseEvent<HTMLElement>, a: number): void => {
-    this._updateColor(ev, updateA(this.state.color, Math.round(a)));
+    this._updateColor(ev, updateA(this.state.color, this._getAlphaOrTransValue(Math.round(a))));
```

Comment:
When using a transparency slider, is the alpha value in the resulting color meant to be interpreted as transparency? If so, you should update the docs for IColor with a note about that.

Reply:
at all times whether using transparency slider or not, the "alpha" hold the actual alpha value only and not the transparency. So no need of updating the docs for IColor.